src/styles.scss

```diff
@@ -51,3 +51,22 @@ body {
   margin: 0;
   font-family: Roboto, 'Helvetica Neue', sans-serif;
 }
+.mat-snack-bar-container{
```

Comment:
All those styles I would move to some other file. This `styles.scss` is a generic file for global and generic styles. Maybe I would move those to `/assets/styles/components/notifications-snackbar.scss` and import this file here or add it to `angular.json`